src/DynamoPackages/PackageLoader.cs

```diff
@@ -310,7 +310,11 @@ public void LoadPackages(IEnumerable<Package> packages)
 
             foreach (var pkg in enumerable)
             {
-                TryLoadPackageIntoLibrary(pkg);
+                // If the pkg is null, then don't load that package into the Library.
+                if (pkg != null)
```

Comment:
This additional check is needed to assert the tests because the testing workflow bypasses the user workflow of downloading the actual package and loading it into Dynamo after being downloaded.

Reply:
In that case, why don't we simply have the check here in one place instead of having it in 2 places, one on the client and one here?

Reply:
An exception was being thrown in the PackageManagerClientViewModel.cs and null reference error message was being shown in the package search dialog. Instead we can show a more clear message to the user. I thought it would be better/cleaner to handle it on the top level as we do not want to make extra function calls. Let me know what you think about it.